app/jobs/calendar_importer/events/squarespace_event.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module CalendarImporter::Events
+  class SquarespaceEvent < Base
+    def initialize(event)
+      @event = event
+    end
+
+    def uid
+      @event['id']
+    end
+
+    def summary
+      @event['title']
+    end
+
+    def description
+      @event['body']
+    end
+
+    def publisher_url
+      @event['url'] + '/' + @event['urlId']
+    end
+
+    # The dates provided are in UTC time which is milliseconds since January 1, 1970, 00:00:00 UTC.
+    # Time.at accepts Unix time which is seconds since January 1, 1970, 00:00:00 UTC.
+    # to account for this we just devide by 1000.
+    def dtstart
+      Time.at(@event['startDate'] / 1000)
+    end
+
+    def dtend
+      Time.at(@event['endDate'] / 1000)
+    end
+
+    def occurrences_between(*)
+      [Dates.new(dtstart, dtend)]
+    end
+
+    def location
+      #get from partner??
+      return nil # TODO: ??? Why are we ignoring the venue for meetup events
+    end
```

Comment:
Why _are_ we not including a location here?

Reply:
This might be linked to some uncertainty around how the location to partner matcher works - not sure. Assume these calendars are relying on the default location strategy. @ivan-kocienski-gfsc can you remember?

Reply:
Moving to new ticket